spec/features/login_spec.rb

```diff
@@ -0,0 +1,62 @@
+require "rails_helper"
+RSpec.feature "Login", type: :feature do
+      
+  let!(:boss) {FactoryBot.create :user}
+  let!(:guest) {FactoryBot.create :user, role: 2}
+  let!(:admin) {FactoryBot.create :user, role: 0}
+  
+  before do
+    visit login_path
+  end
+  
+  feature "Login successfully" do
```

Comment:
feature, context, decribe dùng trong những trường hợp như nào em nhir?

Reply:
+ describe dùng để nhóm các test case theo class, module, method hoặc theo action của controller.
 + context dùng để nhóm các điều kiện của example.
+ feature và scenario là một phần của Capybara chứ không phải RSpec và được sử dụng để kiểm tra chấp nhận. feature tương đương với describe/context,Scenario tương đương với it/example.